benchopt/runner.py

```diff
@@ -265,11 +295,25 @@ def run_one_solver(benchmark, objective, solver, meta, max_runs, n_repetitions,
             # Force the run if needed
             deadline = time.time() + timeout / n_repetitions
 
-            curve_one_rep, status = run_one_to_cvg_cached(
-                benchmark=benchmark, objective=objective, solver=solver,
-                meta=meta_rep, max_runs=max_runs, deadline=deadline,
-                progress_str=progress_str, force=force
-            )
+            if hasattr(solver, "run_all_in_one"):
+                max_iter = int(2 * 1.5 ** (max_runs - 1))
```

Comment:
This should use the value of `RHO`
```suggestion
                max_iter = int(2 * RHO ** (max_runs - 1))
```